Zend/zend_compile.h

```diff
@@ -583,6 +586,10 @@ struct _zend_execute_data {
 #define ZEND_RET_USES_STRICT_TYPES() \
 	ZEND_CALL_USES_STRICT_TYPES(EG(current_execute_data))
 
+#define ZEND_USES_STRICT_OPERATORS() \
+	((EG(current_execute_data) && (EG(current_execute_data)->func->common.fn_flags & ZEND_ACC_STRICT_OPERATORS) != 0) || \
+	 (!EG(current_execute_data) && CG(active_op_array) && (CG(active_op_array)->fn_flags & ZEND_ACC_STRICT_OPERATORS) != 0))
```

Comment:
It's unfortunate that the common run-time case and the rare compile-time cases are combined here.